api/controllers/users.go

```diff
@@ -0,0 +1,64 @@
+package controllers
+
+import (
+	"github.com/uug-ai/facial-access-control/api/database"
+	"github.com/uug-ai/facial-access-control/api/models"
+)
+
+// users godoc
+// @Router /api/users [get]
+// @ID getUsers
+// @Tags users
+// @Summary Get all users
+// @Description Get all users
+// @Success 200 {object} []models.User
+func GetUsers() []models.User {
+	users := database.GetUsers()
+	return users
+}
+
+// user godoc
+// @Router /api/users/{id} [get]
+// @ID getUser
+// @Tags users
+// @Summary Get user by id
+// @Description Get user by id
+// @Param id path int true "User ID"
+// @Success 200 {object} models.User
+func GetUser(id int) models.User {
+	users := database.GetUsers()
```

Comment:
Can we pass in the id to the database methods `GetUser(id)`. When working with a database we can directly query. Move this code to GetUser method of the  file database.